tools/devcluster.yaml

```diff
@@ -48,6 +48,11 @@ stages:
           cache_dir: /tmp/determined-cache
         log:
           level: debug
+        logging:
+          retention:
+            duration: 1
+            cleanup_on_start: true
```

Comment:
why cleanup on start? theuser could just call the CLI to get this behavior, and i don't really expect this being used enough to make it worth the code/docs noise.